### PR TITLE
Added benchmarking code

### LINKED ISSUE #181: Benchmark Polkadex Orderbook
**Context**
Perform different types of load testing of Polkadex Orderbook under different scenarios and create a complete report containing multiple test conditions and outcomes.

 **Tool**
We can use Gatling to perform different types of performance tests on our JSON-RPC based Websockets. Related resource (https://gatling.io/docs/gatling/reference/current/http/websocket/)

**Scenarios**
- [x] Check response time when multiple orders are created.
- [x] Find throughput of Gateway, if multiple create order requests are send.

### PR DESCRIPTION
**Pull request by zktony**
Performs load testing PolkadexTEE Worker using Scala based benchmarking tool i.e. Gatling.
Will close #181

### CODE REVIEW

**Comment by zktony** [comment]
Due to limitation of current web-socket lib as it cant hold more than 100 connections simultaneously, we still dont know current throughput of PolkadexTEE Worker.  As soon as issue #41 closes, I will benchmark PolkadexTEE worker again.

File: polkadex-benchmarking/.bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.5","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/krishnasingh/.sdkman/candidates/java/11.0.11.hs-adpt/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/krishnasingh/.sdkman/candidates/sbt/1.5.4/bin/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/krishnasingh/.sdkman/candidates/sbt/1.5.4/bin/sbt-launch.jar"]}
```
Comment (haerdib) [inline polkadex-benchmarking/.bsp/sbt.json]: should this be uploaded to GH? It seems rather user based than general code update

File: polkadex-benchmarking/src/test/resources/search.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+searchCriterion,searchComputerName
```
Comment (haerdib) [inline polkadex-benchmarking/src/test/resources/search.csv]: Again, rather user specifc. If the user of  this benchmark test needs to adapt this information accordingly, please add a comment somewhere

**Review by haerdib — COMMENTED** [review]
I have not yet worked with scala, so I can't really give a feedback. If you think this needs my approval, I can try to run it locally. But for this, I'd appreciate some more documentation (like prerequistist to have installed, and so on).

But some quick feedback: In this PR there are some files (including binary files, such as .DB_Store) included I'm not sure if they should really be pushed to a remote repo. Are they necessary?

File: polkadex-benchmarking/.bsp/sbt.json (hunk 1 above)
Comment (zktony) [reply polkadex-benchmarking/.bsp/sbt.json]: yeah sorry .. it should not be there

File: polkadex-benchmarking/src/test/resources/search.csv (hunk 2 above)
Comment (zktony) [reply polkadex-benchmarking/src/test/resources/search.csv]: It was part of boilerplate code ... But we dont need it. I removed it

File: polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      .onConnected(exec(ws("sendMessage").sendText(StringBody("""{"jsonrpc": "${jsonrpc}", "method": "${method}", "params": ${params}.toArray, "id": ${id}}"""))
+        .await(20)(ws.checkTextMessage("check1").check(regex(".*")
+          .saveAs("myMessage"))))))*/
+
+  val scene = scenario("testWebSocket")
+    .exec(ws("openSocket").connect("")
+      .onConnected(repeat(100){exec(ws("sendMessage").sendText(message)
+        .await(20)(ws.checkTextMessage("check1").check(regex(".*4,0,0,0.*")
+          .saveAs("myMessage"))))}))
+    // created custom checks for checking my response
```
Comment (simonsso) [inline polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala]: Comments should pressed the code it is describing.

File: polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  val jsonFileFeeder = jsonFile("data.json")
+
+/*val scene = scenario("testWebSocket").feed(jsonFileFeeder)
+    .exec(ws("openSocket").connect("")
+      .onConnected(exec(ws("sendMessage").sendText(StringBody("""{"jsonrpc": "${jsonrpc}", "method": "${method}", "params": ${params}.toArray, "id": ${id}}"""))
+        .await(20)(ws.checkTextMessage("check1").check(regex(".*")
+          .saveAs("myMessage"))))))*/
```
Comment (simonsso) [inline polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala]: If this comment is intended as a comment make it more human readable. If it is for keeping an old version of the code... don't

**Review by simonsso — CHANGES_REQUESTED** [review]
I just had a quick glance and noticed these style errors.

File: polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala (hunk 3 above)
Comment (zktony) [reply polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala]: Code modified

File: polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala (hunk 4 above)
Comment (zktony) [reply polkadex-benchmarking/src/test/scala/computerdatabase/ComputerWorld.scala]: Code Modified

**Review by simonsso — APPROVED** [review]

**Pull request merged**